src/rabbit_amqqueue.erl

```diff
@@ -586,6 +592,13 @@ info_all(VHostPath, Items) ->
     map(list(VHostPath), fun (Q) -> info(Q, Items) end) ++
         map(list_down(VHostPath), fun (Q) -> info_down(Q, Items, down) end).
 
+info_all(VHostPath, Items, Ref, AggregatorPid) ->
+    map(list(VHostPath), fun (Q) -> AggregatorPid ! {Ref, info(Q, Items)} end) ++
```

Comment:
Compiler warning:

src/rabbit_amqqueue.erl:596: Warning: the result of the expression is ignored (suppress the warning by assigning the expression to the _ variable)